packages/flutter/lib/src/material/input.dart

```diff
@@ -257,12 +379,13 @@ class _InputState extends State<Input> {
       );
     }
 
-    return new RepaintBoundary(
-      child: new GestureDetector(
-        behavior: HitTestBehavior.opaque,
-        onTap: () => _rawInputKey.currentState?.requestKeyboard(),
-        child: child
-      )
+    return new GestureDetector(
```

Comment:
This GestureDetector is as big as the entire Input widget, not just its InputField child.